src/pages/veterans/results.jsx

```diff
@@ -25,7 +26,7 @@ class Results extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      year: '2021'
+      year: '2022'
```

Comment:
I'm assuming we hardcode this instead of getting the current year for the case when its a new year, but the race hasn't happened yet. Could we not do the same thing that we do for the `defaultValue` in the render? `this.state = yearOptions[0].value;` 

This way its literally just a one line change, of adding the new year.

For a down the road change, would it make sense just to query the results table and get all years and sort them? That way we can just upload the results without having to make any code change. I'm not sure of the structure, so don't know if that's feasible.

Reply:
Yeah, good idea